src/main/cpp/src/cast_string.cu

```diff
@@ -156,14 +156,14 @@ process_value(bool first_value, T current_val, T const new_digit, bool adding)
  * @param ansi_mode true if ansi mode is required, which is more strict and throws
  */
 template <typename T>
-void __global__ string_to_integer_kernel(T* out,
-                                         bitmask_type* validity,
-                                         const char* const chars,
-                                         size_type const* offsets,
-                                         bitmask_type const* incoming_null_mask,
-                                         size_type num_rows,
-                                         bool ansi_mode,
-                                         bool strip)
+void CUDF_KERNEL string_to_integer_kernel(T* out,
```

Comment:
Should this be

`void CUDF_KERNEL` or `CUDF_KERNEL void` ?

Reply:
Validity wise it doesn't matter. Stylistically it can be argued we should be consistent. I think it's outside the scope of this change to enforce it though. This example started with `void __global__` and ended with `void __global__ static`. This looks odd expanded out, but is valid and does the expected thing. I would prefer to see it as `CUDF_KERNEL void` as well.

Reply:
Oh, I was just replacing all `__global__` with `CUDA_KERNEL` and never noticed this. I agree `CUDA_KERNEL void` is better and consistent. Will fix it.

Reply:
Ah just noticed it has been fixed already in https://github.com/NVIDIA/spark-rapids-jni/pull/2178.